persistence/world/oFunctions.sqf

```diff
@@ -641,10 +647,35 @@ o_loadObjects = {
   };
 
   diag_log format["A3Wasteland - will restore %1 objects", count(_objects)];
+  _realRestoredCounter = 0;
   {
-    _oIds pushBack (_x select 0);
-    [_x] call o_restoreObject;
-  } forEach _objects;
+    _type = _x;
+    //diag_log format ["o_loadObjects type: %1",_type];
+    {
+      private ["_className"];
+      _object_data = call (_x select 1);

```

Comment:
Should add a safety check not proceed if _x is not of CODE type ... you can use if (isCODE(_x))